llvm/include/llvm/TargetParser/X86TargetParser.h

```diff
@@ -121,6 +121,7 @@ enum CPUKind {
   CK_GraniterapidsD,
   CK_Emeraldrapids,
   CK_Clearwaterforest,
+  CK_DiamondRapids,
```

Comment:
CK_Diamondrapids?

Reply:
I prefer your style, too. Let's refine CK_SapphireRapids as well in the future

Reply:
ab477fb